HousingRepairsSchedulingApi/Gateways/AppointmentsGateway.cs

```diff
@@ -0,0 +1,65 @@
+namespace HousingRepairsSchedulingApi.Gateways
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Threading.Tasks;
+    using HACT.Dtos;
+
+    public class AppointmentsGateway : IAppointmentsGateway
```

Comment:
I think it's worth naming this more specifically to represent what it is, i.e. `{Dummy|Static|Relative}AppointmentsGateway`.

A _real_ `AppointmentsGateway` could live alongside this and having the dummy version could easily be used for local development.